### DISCUSSION TITLE
How to disable errors and warnings in the sign column?

### DISCUSSION THREAD

**Discussion by andrewgopher** [VonHeikemen/lsp-zero.nvim#380] [Q&A] [answered]
Hello,

The title is self-explanatory. Note that I don't want to disable the sign column because I also use git gutter.

Any help would be appreciated.

**Comment by VonHeikemen** [comment, marked as answer]
You can disable them using `vim.diagnostic.config`.

```lua
vim.diagnostic.config({
  signs = false
})
```
